docs/docs/known-issues.md

```diff
@@ -4,8 +4,16 @@ sidebar_position: 12
 
 # Known Issues
 
-Encord Active is in beta phase, therefore, errors and unexpected behaviours may sometimes be encountered.
+Encord Active is currently in the beta phase, which means that occasional errors and unexpected behaviours may occur.
 
-- In some projects/metrics we have observed that the UI freezes and eventually crashes in the browser.
+- We have observed instances where the UI freezes and eventually crashes in certain projects/metrics when accessed through the browser.
```

Comment:
Shall we remove this?

Reply:
I let it there so we have something not ok in the list just in case someone finds something wrong and goes there first is not an empty list. Appreciated if you come up with some potential error that's more likely to happen now and I can change it for that one.